src/tests/JIT/Directed/tailcall/more_tailcalls.il

```diff
@@ -5,11 +5,7 @@
 
```

Comment:
There is a more_tailcalls.cs that was used to automatically generate this file. Can you please make sure they are kept in sync?

Reply:
@jakobbotsch Sure, thanks for bringing it up!

Reply:
Which process are we using to generate the file? msbuild to compile the .ilproj and ILSpy to generate the C#?